framework/src/multiapps/MultiApp.C

```diff
@@ -182,8 +182,13 @@ MultiApp::validParams()
       "List of subapp postprocessors to use coupling "
       "algorithm on during Multiapp coupling iterations");
 
+  params.addDeprecatedParam<bool>(
+      "clone_master_mesh",
+      false,
+      "True to clone master mesh and use it for this MultiApp.",
+      "clone_master_mesh is deprecated, use clone_parent_mesh instead");
   params.addParam<bool>(
-      "clone_master_mesh", false, "True to clone master mesh and use it for this MultiApp.");
+      "clone_parent_mesh", false, "True to clone master mesh and use it for this MultiApp.");
```

Comment:
`clone master mesh` => `clone parent mesh`.

Reply:
Should have been done already. I ll double check